TelemetryExporter.UI/Views/SelectWidgets.xaml.cs

```diff
@@ -153,10 +155,18 @@ private async void ExportBtn_Clicked(object sender, EventArgs e)
 
         try
         {
-            await exporter.ExportImageFramesAsync(
+            // This will generate a short unique string. Differentiate multiple exports, avoid file exist.
+            // It is fine for generating one time. .Multiple generating ids for less than millisecond that depends
+            // on dateTime will not work
+            // https://stackoverflow.com/a/41723783/6142097
+            string uniqueIdentifier = DateTime.UtcNow.Ticks.ToString("x");
```

Comment:
```suggestion
            // This will generate a short unique string. Differentiate multiple exports, avoid check for existing file.
            // It is fine for generating one time. Multiple generating ids for less than millisecond that depends
            // on dateTime will not work
            // https://stackoverflow.com/a/41723783/6142097
            string uniqueIdentifier = DateTime.UtcNow.Ticks.ToString("x");
```